index.js

```diff
@@ -11,40 +11,37 @@ function instrument(options) {
   var instrumenter = new Instrumenter.Instrumenter();
   var captured = false;
 
-  function transform(file) {
-    // If if doesnt match the pattern dont instrument it
+  return through.obj(function(row, enc, next) {
+    if (row.file.indexOf("node_modules") !== -1) {
```

Comment:
Well, your previous implementation used a transform. Unless a transform is explicitly marked as `global`, Browserify doesn’t apply it to stuff in `node_modules`. This case is here to retain this behavior. If I don’t do this, it’s breaking the existing behavior.